modeling.py

```diff
@@ -0,0 +1,146 @@
+import nimfa
+import pandas as pd
+from nfldb_tables import NFLFrames
+from scipy.sparse import csr_matrix as csrm
+
+
+class PositionNMF(object):
```

Comment:
I don't see what this class does that we couldn't just do with the DataFrame class.

Why not have the class you've called PositionNMFFactory return a DataFrame and cut this class entirely?